spec/services/vine_api_service_spec.rb

```diff
@@ -0,0 +1,83 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+
+RSpec.describe VineApiService do
+  subject(:vine_api) { described_class.new(api_key: vine_api_key, jwt_generator: jwt_service) }
+
+  let(:vine_api_url) { "https://vine-staging.openfoodnetwork.org.au/api/v1" }
+  let(:vine_api_key) { "12345" }
+  let(:jwt_service) { VineJwtService.new(secret:) }
+  let(:secret) { "my_secret" }
+
+  before do
+    allow(ENV).to receive(:fetch).and_call_original
+    allow(ENV).to receive(:fetch).with("VINE_API_URL").and_return(vine_api_url)
```

Comment:
I just had a quick look to see if there's a better way to mock environment variables... but couldn't find one.